perf/perf_stress_ng.py

```diff
@@ -243,10 +243,12 @@ def tearDown(self):
         removes the log files and collects the dmesg data
         :param: none
         """
-
-        file_name = "rm -rf /tmp/stdout /tmp/stderr /tmp/data* /tmp/stressng_output*"
-        try:
-            os.remove(file_name)
-        except OSError:
-            self.log.warn("Files do not exist")
+        files_to_remove = ["/tmp/stdout", "/tmp/stderr", "/tmp/data*",
+                           "/tmp/stressng_output*"]
```

Comment:
Going through the code i could see, almost everywhere `/tmp/*` file is being used to dump data, can a better way be to use `self.workdir` for dumping logs and clearing those files at last combinly, as using self.workdir is more clean and is better way to handle situations.
what do you think @disgoel.

Reply:
I experimented with substituting /tmp with self.workdir, but encountered unexpected behaviour and increased code complexity. It seems we might need to address this issue at a later time when we can find a more effective solution.